Common/DarwinMemoryStickManager.mm

```diff
@@ -0,0 +1,98 @@
+//
+//  DarwinMemoryStickManager.mm
+//  PPSSPP
+//
+//  Created by Serena on 20/01/2023.
+//
+
+#include "ppsspp_config.h"
+#include "Core/Config.h"
```

Comment:
`Common/` should not include `Core/` files.  Should this just be under `ios/`?

The dependency chain should be that Common is depended on by other code, by Core, GPU, etc. are not dependencies of Common.  Otherwise it can be circular.

-[Unknown]

Reply:
The code is shared by both macOS and iOS, so it wouldn’t make any sense to place it under `ios/`, would the file be appropriate to place under `Core/`?

Reply:
Ah right, sorry.  Yeah, that sounds fine.  Or even under UI, since all consumers are in UI/.  Could be either.

-[Unknown]